src/main/resources/avro/metadata.avdl

```diff
@@ -122,10 +122,25 @@ For data accessors, they're a simple way to scope exploration and analysis
 what's the distribution of that result in the data from our project?)   
 */
 record Dataset {
-  /** The dataset UUID. This is globally unique. */
+  /**
+  The dataset's id. This is the local/legacy/generic id.
```

Comment:
I would leave out the second sentence in this comment. We will (hopefully) have a high-level document explaining the distinction between an ``id`` (a server generated string that is unique and persistent within a particular implementation, but not portable across implementations), and a ``name`` (a user specified string which is intended to provide a more human friendly handle for objects).